# Question 6
-----

$$f(x) = ax^2 + bx$$

$$\int_{0}^{2} ax^2 + bx \ dx = 1$$

$$[\frac{ax^3}{3} + 0.5bx^2]_{0}^{2} = 1$$

$$\frac{8a}{3} + 2b = 1$$

$$8a + 6b = 3$$

$$\int_{0}^{2} ax^3 + bx^2 \ dx = 1.25$$

$$[0.25ax^4 + \frac{bx^3}{3}]_{0}^{2} = 1.25$$

$$4a + \frac{8b}{3} = 1.25$$

$$12a + 8b = 3.75$$

solving for b,

$$8a = 3 - 6b$$

$$a = \frac{3 - 6b}{8}$$

$$\frac{12(3 - 6b)}{8} + 8b = 3.75$$

$$12(3 - 6b) + 64b = 30$$

$$36 - 72b + 64b = 30$$

$$-8b = -6$$

$$b = 0.75$$

solving for a,

$$a = \frac{3 - 6(0.75)}{8}$$

$$a = \frac{-1.5}{8}$$

$$a = -0.1875$$

-----
b)

$$E(X^2) = \int_{0}^{2} -0.1875x^4 + 0.75x^3 \ dx$$

$$E(X^2) = [-0.0375x^5 + 0.1875x^4]_{0}^{2}$$

$$E(X^2) = 1.8$$

$$Var(X) = 1.8 - (1.25^2)$$

$$Var(X) = 0.2375$$

-----
c)

$$f(x) = -0.1875x^2 + 0.75x$$

$$F(x) = -0.0625x^3 + 0.375x^2$$

$$0.375x^2 - 0.0625x^3 = 0.5$$

Using an exhaustive solution:

In [12]:
x = 0
fx = 0
while fx < 0.5:
    fx = (0.375 * pow(x, 2)) - (0.0625 * pow(x, 3))
    x = x + 0.01

print(x)

1.320000000000001


Hence, the median occurs at x = 1.32

Since $0 \le x \le 2$, mode = 2